stan/math/prim/mat/prob/gaussian_dlm_obs_lpdf.hpp

```diff
@@ -109,7 +105,8 @@ gaussian_dlm_obs_lpdf(
   check_pos_definite(function, "C0", C0);
   check_finite(function, "C0", C0);
 
-  if (y.cols() == 0 || y.rows() == 0)
+  T_lp lp(0.0);
+  if (size_zero(y))
```

Comment:
In the last function, these just returned `0` (which will get promoted) and the `lp` variable was defined later.  I liked it the way it was in the last function, but either way, it should probably be consistent.

Reply:
Agreed, fixed to return 0 and define `lp` later.